.github/workflows/test.yaml

```
@@ -8,6 +8,7 @@ jobs:
   test:
     env:
       WANT_STATUS: 103 # ExitCodeHigh
+      WANT_CONFIG: testdata/lava.yaml
```

I would rename `WANT_CONFIG` to `CONFIG` because you are using the var in `with` and looks weird.

```suggestion
      CONFIG: testdata/lava.yaml
```